src/plugins/menu_qrcode.py

```diff
@@ -1,18 +1,25 @@
 # -*- coding: utf-8 -*-
 """
+src/plugins/menu_qrcode.py
+==========================
+
 A menu plugin showing QR-Code for bitmessage address in modal dialog.
 """
-
-from PyQt4 import QtGui, QtCore
-import qrcode
+# pylint: disable=import-error
 
 from pybitmessage.tr import _translate
+from PyQt4 import QtCore, QtGui
+
+import qrcode
```

Comment:
Wrong order: PyQt4 and qrcode are [third party imports](https://www.python.org/dev/peps/pep-0008/#imports).